packages/teams-js/src/public/tasks.ts

```diff
@@ -55,34 +70,36 @@ export namespace tasks {
     dialog.submit(result, appIds);
   }
 
-  function getDialogInfoFromTaskInfo(taskInfo: TaskInfo): TaskInfo {
-    const dialogHeight =
-      taskInfo.height && typeof taskInfo.height !== 'number'
-        ? getDialogDimensionFromTaskModuleDimension(taskInfo.height)
-        : (taskInfo.height as number);
-    const dialogWidth =
-      taskInfo.width && typeof taskInfo.width !== 'number'
-        ? getDialogDimensionFromTaskModuleDimension(taskInfo.width)
-        : (taskInfo.width as number);
-    const dialogInfo: TaskInfo = {
+  export function getUrlDialogInfoFromTaskInfo(taskInfo: TaskInfo): UrlDialogInfo {
+    const urldialogInfo: UrlDialogInfo = {
       url: taskInfo.url,
-      card: taskInfo.card,
-      height: dialogHeight,
-      width: dialogWidth,
+      size: {
+        height: taskInfo.height,
+        width: taskInfo.width,
+      },
       title: taskInfo.title,
       fallbackUrl: taskInfo.fallbackUrl,
-      completionBotId: taskInfo.completionBotId,
     };
-    return dialogInfo;
+    return urldialogInfo;
   }
 
-  function getDialogDimensionFromTaskModuleDimension(taskModuleDimension: TaskModuleDimension): TaskModuleDimension {
-    if (taskModuleDimension === TaskModuleDimension.Large) {
-      return TaskModuleDimension.Large;
-    } else if (taskModuleDimension === TaskModuleDimension.Medium) {
-      return TaskModuleDimension.Medium;
-    } else {
-      return TaskModuleDimension.Small;
-    }
+  export function getBotUrlDialogInfoFromTaskInfo(taskInfo: TaskInfo): BotUrlDialogInfo {
+    const botUrldialogInfo: BotUrlDialogInfo = {
+      url: taskInfo.url,
+      size: {
+        height: taskInfo.height,
+        width: taskInfo.width,
+      },
+      title: taskInfo.title,
+      fallbackUrl: taskInfo.fallbackUrl,
+      completionBotId: taskInfo.completionBotId,
+    };
+    return botUrldialogInfo;
   }
 }
+
+export function getDefaultSizeIfNotProvided(taskInfo: TaskInfo): TaskInfo {
+  taskInfo.height = taskInfo.height ? taskInfo.height : TaskModuleDimension.Medium;
```

Comment:
I think in the doc we shared with partners we mentioned that we would default to small. I don't think it's particularly important that we stick to that, but why the change?

Reply:
Oh, I am sorry, I missed that. I will change it to small